Support for python 3.12 (#24)
WayScience/pairwise_compare: README.md

If you choose to interact with another component of the tool, then there will be less input validation safeguards available.

## Installation

> In Python 3.10 to 3.12, inclusive, you can install this tool with either of the following commands:<br>

Reviewer: ```suggestion
For Python versions 3.10 through 3.12 (inclusive), you can install this tool with either of the following commands:<br>
```
Optional nit-pick, I found that above was a bit easier to read and less technical, but your version also works too!

Author: I like it
